rust/kcl-lib/src/simulation_tests/kcl_samples.rs

```diff
@@ -184,3 +235,152 @@ fn kcl_samples_outputs(filter: Option<&str>) -> Vec<String> {
 
     outputs
 }
+
+const MANIFEST_FILE: &str = "manifest.json";
+const COMMENT_PREFIX: &str = "//";
+
+#[derive(Serialize, Deserialize, Debug)]
+#[serde(rename_all = "camelCase")]
+struct KclMetadata {
+    file: String,
+    path_from_project_directory_to_first_file: String,
+    multiple_files: bool,
+    title: String,
+    description: String,
+}
+
+// Function to read and parse .kcl files
+fn get_kcl_metadata(project_path: &Path, files: &[String]) -> Option<KclMetadata> {
+    // Find primary kcl file (main.kcl or first sorted file)
+    let primary_kcl_file = files
+        .iter()
+        .find(|file| file.contains("main.kcl"))
+        .unwrap_or_else(|| files.iter().min().unwrap())
```

Comment:
In `kcl_samples_inputs()` in this file, I made it fall back to the .kcl file with the same name as the directory. Maybe that should get updated?

Reply:
oh thats a bad practice entirely ill update any time we do that, we should always use main.kcl

Reply:
oh we don't do that anywhere so we are good, im going to remove your fall back just so people dont do that haha